### PR TITLE
Inc cdt name cos7

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.
  * [x] added `httpdate`

- contains #14 (and probably #13)
  - i still haven't run down what the underlying issue is (probably my fault, of course), but if it's just licenses blocking it...

### CODE REVIEW

**Comment by bollwyvl** [comment]
cc @jakirkham

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ source:
   sha256: 4809d4a181e33796488d8481120427c717e54e3deedccb253a3bdbf2dc931c76
 
 build:
-  number: 0
+  number: 1
```
Comment (bollwyvl) [inline recipe/meta.yaml:19]: I assume this should be bumped?

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -2,3 +2,5 @@ cuda_compiler_version:                    # [linux64]
   - None                                  # [linux64]
 docker_image:                             # [linux64]
   - condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                 # [linux64]
+  - cos7                                  # [linux64]
```
Comment (bollwyvl) [inline recipe/conda_build_config.yaml:6]: So I assume this is what I missed?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -136,6 +136,8 @@ about:
     - library_licenses/http-LICENSE-MIT
     - library_licenses/httparse-LICENSE-APACHE
     - library_licenses/httparse-LICENSE-MIT
+    - library_licenses/httpdate-LICENSE-APACHE
+    - library_licenses/httpdate-LICENSE-MIT
```
Comment (bollwyvl) [inline recipe/meta.yaml:140]: Gives me a bit of the willies that we pulled a different novel depdendency from the same source tarball...

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/conda_build_config.yaml:6]: Yeah that's it.

Edit: We've also added this to [the docs]( https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7 ) recently.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:19]: Shouldn't be needed unless there are other changes (like packaging license files as I'm now seeing). Though it doesn't hurt to have it.

File: recipe/meta.yaml (hunk 3 above)
Comment (jakirkham) [reply recipe/meta.yaml:140]: Yeah we would need a build number bump to include the new license files. So the change above makes sense.

**Comment by bollwyvl** [comment]
Alright, ducks in order based on above comments... will merge if it comes up green! Otherwise will be a few hours before I can monkey with it more...

**Comment by jakirkham** [comment]
Thanks Nick! 😄

**Comment by bollwyvl** [comment]
Thank _you_ for catching my error :blush: and helping get the fix shipped :checkered_flag: ! Does anything i _already_ ship with the wrong settings need to be re-released and/or marked broken?

**Comment by jakirkham** [comment]
Nope we should be good. There’s no error on your part. This is just a new setting we realized we needed to add.

**Pull request merged**